kuma/contributions/jinja2/contributions/thank_you.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'landing/base_promote.html' %}
```

Comment:
This is the same as the ``base.html`` template, except it adds a class "section-community" to the ``<body>``. I'd like to change to ``base.html``, we don't appear to be using any customizations for that class.